src/LazyOperations/CachedMinkowskiSumArray.jl

```diff
@@ -140,7 +139,7 @@ If the direction has norm zero, the result depends on the summand sets.
 
 The result is cached, i.e., any further query with the same direction runs in
 constant time.
-When sets are added to the caching Minkowski sum, the query is only performed
+When sets are added to the cached Minkowski sum, the query is only performed
```

Comment:
in several places we had `caching` instead of `cached`. i think we should use only one name (?) so `cached` is better since it is part of the type's name.